server/@types/dto/index.d.ts

```diff
@@ -174,4 +175,34 @@ declare module 'dto' {
     challenges?: Array<ChallengeType>
     strengths?: Array<StrengthType>
   }
+
+  /**
+   * DTO collating the Additional Learning Needs (ALN) and Learning Difficulties and Disabilities (LDD) assessments that have been recorded in Curious
+   */
+  export interface CuriousAlnAndLddAssessmentsDto {
+    lddAssessments: Array<CuriousLddAssessmentDto>
+    alnAssessments: Array<CuriousAlnAssessmentDto>
+  }
+
+  /**
+   * DTO representing a Learning Difficulties and Disabilities (LDD) assessment that has been recorded in Curious
+   */
+  export interface CuriousLddAssessmentDto {
+    prisonId: string
+    rapidAssessmentDate: Date
```

Comment:
These date fields could perhaps do with better names, or at least an understanding of what a "rapid" and "in depth" assessment are; and why a prisoner might have one or the other (or both? 🤷‍♂️ )